data.sql

```diff
@@ -13,12 +13,22 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALU
 ('Blossom', '1998-10-13', 3, true, 17),
 ('Ditto', '2022-05-14', 4, true, 22);
 
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Jennifer Orwell', 19),
```

Comment:
- [ ] Please, consider fixing the issue marked by the previous reviewer, about removing the doubled entry `('Jennifer Orwell', 19),`

Reply:
Done